modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
Agreed, global variables should be worked on in another PR, which will include work on lib-jitsi-meet as well.